### DISCUSSION TITLE
New AIO instance

### DISCUSSION THREAD

**Discussion by Poupoussdu412** [nextcloud/all-in-one#4622] [Questions] [answered]
Hi,

When i'm trying to launch the new aio instance

NOTICE: PHP message: It seems like the ip-address of ***************** is set to an internal or reserved ip-address. (It was found to be set to '192.168.2.33')
</html>nter>openresty</center>1></center>d>nnection attempt to "********************:443" was: <html>
NOTICE: PHP message: Expected was: 9d8ad6c54fa3c9aa6ac459cd7be3fd470061b119748a1d29

Ofc it's himself, somebody has an idea ?

**Comment by mkultracreative** [comment, marked as answer]
# Mission: find and locate the stalker of that IP that's too jealous to share...
### then killemall. Or `killall` to be specific. But first...
**

- [ ] Your network shows your IP address  right in the DNS settings. Stop using an IP that's already reserved on another instance / computer.

- [ ] In the case where _you_ added that IP to your own machine, then this error will also occur when asking for a new promise against an IP that's already engaged to another thread. (For example, if you started a Postgres container, it will hold first rights to refusal until implicit release of the mounted container / or in some rare cases even the database itself could be awaiting in a promise state and never receiving a final response) which will cause a problem that's basically invisible). 

Solution:
1. killall apps and PID's lingering, in any way, to that computer or server.
2. note the current IP in the DNS for backup then change it to something random enough it can't be a fallback
_(ie avoid just going from just 0.1.20 to 0.1.1.21) -- just to avoid some cases where a process will infer a fallback IP using a rollup query)._

- [ ] Change your repo's environment to point to the new server (make sure you're cross-validating both your internal and external ports for each IP address in your networks whitelist 

- [] if you never did that, go do it now 😂

- rebuild your project installs and start the dev server again

Come back and let me know how it feels to be the king of getting shit done today. ![image](https://github.com/nextcloud/all-in-one/assets/63879450/ae968e0e-6854-4388-a926-74ea553a6b08)
